provider/store.go

```diff
@@ -4,9 +4,12 @@
 package provider
 
 import (
+	"math/big"
+
 	"github.com/ChainSafe/fil-secondary-retrieval-markets/shared"
 )
 
 type RetrievalProviderStore interface {
 	Has(params shared.Params) (bool, error)
+	GetSize(params shared.Params) (*big.Int, error)
```

Comment:
I'm not sure size is still required. Please check out the latest retrieval params, we may need to update ours https://github.com/filecoin-project/go-fil-markets/blob/c7bd334e322c387e14e7f00e745f02d6cb256c26/retrievalmarket/types.go#L234-L241

Reply:
I've updated the `QueryResponse` to have `PricePerByte` instead of `Total`, let me know what you think